Firebase.cpp

```diff
@@ -28,11 +28,18 @@ Firebase& Firebase::auth(const String& auth) {
 }
 
 String Firebase::get(const String& path) {
-  return sendRequest("GET", path);
+  sendRequest("GET", path);
+  return getBody();
 }
 
 String Firebase::push(const String& path, const String& value) {
-  return sendRequest("POST", path, value);
+  sendRequest("POST", path, value);
+  return getBody();
+}
+
+bool Firebase::remove(const String& path) {
```

Comment:
I feel like we should return `void` here, to stay inline with the current way we have to check for error with `error()`.